bin/fusioninventory-injector

```diff
@@ -33,14 +33,20 @@ GetOptions(
 $OUTPUT_AUTOFLUSH = 1;
 pod2usage(-verbose => 0, -exitstatus => 0) if $options->{help};
 
-if ($options->{stdin}) {
-    loadstdin();
-} elsif ($options->{file}) {
-    loadfile($options->{file});
-} elsif ($options->{directory}) {
-    loaddirectory($options->{directory});
-} else {
-    pod2usage();
+eval {
+    if ($options->{stdin}) {
+        loadstdin();
+    } elsif ($options->{file}) {
+        loadfile($options->{file});
+    } elsif ($options->{directory}) {
+        loaddirectory($options->{directory});
+    } else {
+        pod2usage();
+    }
+};
+if ($@) {
+    print $@;
```

Comment:
`die "$@\n";` should be better here. The "\n" avoids to show the line where you die and we don't need it here.